### PR TITLE
Score, start button, timer(wip), and q's

### PR DESCRIPTION
**Pull request by paramighty**
Please merge my amazing code.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -177,3 +185,62 @@ function pickAFlag() {
...
+divCurrentScore.id = "currentScore";
+const scoreText = document.createTextNode("Total Score: 0");
+
+divCurrentScore.appendChild(scoreText);
+document.body.insertBefore(
+  divCurrentScore,
+  document.getElementById("container")
+);
+document.getElementById("currentScore").style.fontSize = "50px";
+
```
Comment (staffanosp) [inline script.js:201]: I would have created this in a "updateScore" function to bind state and ui.

Comment (paramighty) [reply script.js:201]: SattaFuckUP Staffan. MY WAY IS HIGH WAY.

**Review by paramighty — COMMENTED** [review]
DO WHAT YOU GOTTA DO

File: script.js (hunk 1 above)
Comment (staffanosp) [reply script.js:201]: Great stuff!

**Pull request merged**